specifyweb/frontend/js_src/lib/components/Attachments/Viewer.tsx

```diff
@@ -98,7 +98,7 @@ export function AttachmentViewer({
         {originalUrl === undefined ? (
           loadingGif
         ) : type === 'image' ? (
-          <img alt={title} src={originalUrl} />
+          <img alt={title} className="max-h-[70vh]" src={originalUrl} />
```

Comment:
This is not a great fix. Please avoid hardcoded sizing whenever possible.
A nice fix would make the image occupy the full height of the container, regardless of container size (your fix assumes the container is always near the full screen in size)

Here is a preview of what desired behavior may look like - try making it look like that (except for the image dimensions distortion - which can be fixed using object-fit CSS property) - not showing you the code to leave it as an exercise

https://user-images.githubusercontent.com/40512816/233509885-7c0d0f31-72e0-451d-ae88-2180b8c39057.mp4